bootstrap/src/main/java/io/jenkins/jenkinsfile/runner/bootstrap/Bootstrap.java

```diff
@@ -5,6 +5,7 @@
 import org.kohsuke.args4j.CmdLineParser;
 import org.kohsuke.args4j.Option;
 
+import javax.annotation.CheckForNull;
```

Comment:
You added FIndbugs but the annotation is the one from javax?

Reply:
Yes. It comes from `com.google.code.findbugs`